patterns/vsi/variables.tf

```diff
@@ -146,6 +146,14 @@ variable "vsi_per_subnet" {
   default     = 1
 }
 
+variable "user_data" {
```

Comment:
@ocofaigh do we need to be more specific in this description for how this new map is to be used? The usage would be in the form of:

```
{ a_vpc_name = { user_data = <the data> }, another_vpc_name = { user_data = <the data> }}
```
where the vpc names match what is in the `vpcs` input, typically "workload" and "management". I don't know if this map format is obvious to end user?

Reply:
The description is pointing to a cloud doc which explains more (its not yet published)